### PR TITLE
CI: add concepts exercises to tests

### PR DESCRIPTION
**Pull request by ynfle**
The second commit (6ef5d42b93e0bf466e491a96ea5a748ea913538b) is the one that needs to be reviewed. The first commit (eea79e407058f2d33cdffd91bded3144728ff0be) is just there to be able to use the test file that isn't empty that will be added soon (https://github.com/exercism/nim/pull/339). Without this, the CI fails ([see this run](https://github.com/exercism/nim-test-runner/runs/5559905902?check_suite_focus=true)) because the manipulated test file is invalid thus compilation fails.
 - [x] Merge https://github.com/exercism/nim/pull/339
 - [x] Remove eea79e407058f2d33cdffd91bded3144728ff0be

**EDIT**: Updated description

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -49,7 +49,10 @@ jobs:
     - uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # 2.4.0
 
     - name: Download all `exercism/nim` exercises
-      run: git clone --depth 1 https://github.com/exercism/nim.git
+      run: |
+        git clone https://github.com/exercism/nim.git
+        cd nim
+        git checkout 8710920
```
Comment (ErikSchierboom) [inline .github/workflows/ci.yml:55]: I assume this is temporary? If so, maybe add a comment explaining why this is being done?

**Review by ErikSchierboom — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (ynfle) [reply .github/workflows/ci.yml:55]: Ya, that is what I meant by the first comment on the PR.

There are 2 commits. The first one is to specifically checkout my commit that adds tests for the lasagna exercise and the second adds the concept exercises to the docker container tests. Without the a non-empty test file, the test runner errors out (I'm not 100% percent sure, but I'm pretty sure we insert some unit test output formatter setup code before the first test suite, but there is no test suite so we attempt to close the formatter file stream which errors out), thus making the CI fail.

Comment (ynfle) [reply .github/workflows/ci.yml:55]: Updated description

Comment (ErikSchierboom) [reply .github/workflows/ci.yml:55]: Okay, great

Comment (ynfle) [reply .github/workflows/ci.yml]: Commit removed as https://github.com/exercism/nim/pull/339 has been merged

**Pull request merged**